dashboard/app/controllers/aichat_controller.rb

```diff
@@ -0,0 +1,34 @@
+class AichatController < ApplicationController
```

Comment:
Yeah, my thought was it would be good to decouple the name of the controller with the specific endpoint it's hitting (in case we for some reason switch from SageMaker back to OpenAI or something else). I do think moving SageMaker-specific code to its own unit will be good though.